src/test/scala/GameSpec.scala

```diff
@@ -1,12 +1,12 @@
-import rps.service.{GameService}
+import rps.service.{GameServiceImpl}
 import rps.repository.{InMemoryGameRepository}
 import rps.model.{Move, Result}
 import Move._, Result._
 import org.scalatest._
 
 class GameSpec extends FlatSpec {
   implicit val repo = InMemoryGameRepository()
```

Comment:
This is the dual of my comment above: don't make this implicit, so the dependency between `gameService` and `gameRepository` is more obvious to read.

Reply:
To re-connect with previous discussions, this is where you would mock `gameRepository`, if you need to.